topics/transcriptomics/tutorials/scrna-raceid/tutorial.md

```diff
@@ -174,23 +178,24 @@ We can see this for ourselves by extracting the headers, and reformatting them t
 > ### {% icon question %} Questions
 >
 > 1. How many unique cell phenotypes were identified in the cell headers?
-> 2. Which cell phenotype is least represented in the count matrix?
+> 1. Which cell phenotype is least represented in the count matrix?
 >
 > > ### {% icon solution %} Solution
 > >
 > > 1. There are 5 types of cells in our count matrix: *I5d*, *II5d*, *III5d*, *IV5d*, and *V5d*.
-> > 2. There are only 48 *IV5d* cells compared to the other types which have 95 or 96.
+> > 1. There are only 48 *IV5d* cells compared to the other types which have 95 or 96.
 > {: .solution}
 >
 {: .question}
 
-With these types already labelled in the header of our data, we can validate the clustering that we will perform later. Ideally, the cells described by these 5 different labels should cluster into 5 separate clusters, with varying degrees of proximity to one another.
+With these types already labelled in the header of our data, we can validate the clustering that we will perform later. Ideally, the cells described by these 5 different labels should group into 5 separate clusters, with varying degrees of proximity to one another.
 
 ## Inspecting the Quality of the Count Matrix
 
-Low quality cells and genes are sometimes caught at the preprocessing stage and removed, but sometimes more filtering is required to remove unwanted noise from the data.
+Low quality cells and genes are often caught at the preprocessing stage and removed, but sometimes more filtering is required to remove unwanted noise from the data.
+
+A gene with a low number of total counts across multiple cells might be differentially expressed across those cells, however differential gene expression and log fold change do not necessarily denote significant change. 1 count in CellA and 10 counts in CellB yields an LFC of 10, but is not as significant as an LFC of 10 from 100 counts in CellA vs 1000 counts in CellB.
```

Comment:
This appears to be the only mention of LFC - I don't think the abbreviation is required, unless you think it might be useful for other tutorials?

Reply:
Looks good